packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,19 @@
+:local {
+  /* stylelint-disable selector-class-pattern  */
+  .card {
+    background-color: var(--terra-card-background-color, #fff);
+    border-radius: var(--terra-card-border-radius, 3px);
+    box-shadow: var(--terra-card-box-shadow, 0 1px 1px 0 #1c1f21);
+    overflow: hidden;
```

Comment:
This was my original comment that had him add this.
https://github.com/cerner/terra-core/pull/727#discussion_r133099940
Its based off our previous experience using building this with the terra-status element
With overflow hidden
![screen shot 2017-08-15 at 3 50 08 pm](https://user-images.githubusercontent.com/8029578/29335857-a065780e-81d1-11e7-9732-d6d2dfe72317.png)
Without overflow hidden
![screen shot 2017-08-15 at 3 50 15 pm](https://user-images.githubusercontent.com/8029578/29335861-a4d57542-81d1-11e7-90cb-56d153d05562.png)
https://r-wellness-components.devhealtheintent.com/Card-Component-Example
I know this link isn't public, but i'll put it here incase you want to see. Let us know if there is a better way to approach this. 
Thanks